shared/redux/rtk-apis/assignments/assignments.api.ts

```diff
@@ -22,8 +22,16 @@ const assignmentsApi = projectApi.injectEndpoints({
       },
       invalidatesTags: ["Assignments"],
     }),
+
+    getClassroomAssignments: builder.query<AssignmentsResponseDto[], number>({
+      query: (classroomId) => ({
+        url: `/classrooms/${classroomId}/assignments`,
+        method: "GET",
+      }),
+      providesTags: ["Assignments"],
```

Comment:
Is this sufficient to achieve optimal validation? Wouldn't updating an assignment of any classroom lead to invalidation of assignments in all classrooms?